frontend/app/services/GuardianContentService.scala

```diff
@@ -66,13 +59,13 @@ class GuardianContentService(actorSystem: ActorSystem, executionContext: Executi
   private val contentApiPeriod = 30.minutes
 
   val masterclassContentTask = ScheduledTask[Seq[MasterclassData]](
-    "GuardianContentService - Masterclass content", Nil, 2.seconds, contentApiPeriod)(masterclasses)
+    "GuardianContentService - Masterclass content", Nil, 2.seconds, contentApiPeriod)(masterclasses())
 
   val contentTask = ScheduledTask[Seq[Content]](
-    "GuardianContentService - Content with Eventbrite reference", Nil, 1.millis, contentApiPeriod)(eventbrite)
+    "GuardianContentService - Content with Eventbrite reference", Nil, 1.millis, contentApiPeriod)(eventbrite())
```

Comment:
these are now calling the new recursive function with default args, rather than the old enumerator.  This is still going to be reevaluated on every call, as it should (otherwise the events list would only update on deploy!)